extensions/tags/js/src/forum/addTagComposer.js

```diff
@@ -62,9 +62,14 @@ export default function () {
     const chosenSecondaryTags = chosenTags.filter(tag => tag.position() === null);
     const selectableTags = getSelectableTags();
 
-    if ((!chosenTags.length
-          || (chosenPrimaryTags.length < app.forum.attribute('minPrimaryTags'))
-          || (chosenSecondaryTags.length < app.forum.attribute('minSecondaryTags'))
+    const minPrimaryTags = parseInt(app.forum.attribute('minPrimaryTags'));
+    const minSecondaryTags = parseInt(app.forum.attribute('minSecondaryTags'));
+    const maxPrimaryTags = parseInt(app.forum.attribute('maxPrimaryTags'));
+    const maxSecondaryTags = parseInt(app.forum.attribute('maxSecondaryTags'));
+
+    if (((!chosenTags.length && maxPrimaryTags !== 0 && maxSecondaryTags !== 0)
+          || (chosenPrimaryTags.length < minPrimaryTags)
+          || (chosenSecondaryTags.length < minSecondaryTags)
         ) && selectableTags.length) {
```

Comment:
Does this not work, too? I think only removing the check that no tags are selected is enough.

```suggestion
    const minPrimaryTags = parseInt(app.forum.attribute('minPrimaryTags'));
    const minSecondaryTags = parseInt(app.forum.attribute('minSecondaryTags'));
    const maxPrimaryTags = parseInt(app.forum.attribute('maxPrimaryTags'));
    const maxSecondaryTags = parseInt(app.forum.attribute('maxSecondaryTags'));

    if (selectableTags.length && (chosenPrimaryTags.length < minPrimaryTags || chosenSecondaryTags.length < minSecondaryTags
    )) {
```

Reply:
Sorry I forgot about this PR 😞 

Your change makes sense but it misses another scenario, the current behavior which we don't want to lose is that: If the maximum is not nil, and the user chooses no tags (with min being 0) the tag selection modal should show up because they can select something.